wazo_call_logd/plugins/cdr/schemas.py

```diff
@@ -173,6 +174,15 @@ def _populate_tags_field(self, data, **kwargs):
         return data
 
 
+class UserCDRSchema(CDRSchema):
+    recordings = fields.Nested(
+        'RecordingSchema',
+        many=True,
+        default=[],
+        exclude=('conversation_id', 'storage_path'),
```

Comment:
Maybe we want to use same syntax for `tags` exclusion and these fields (use `exclude` vs redefine schema)
https://github.com/wazo-platform/wazo-call-logd/blob/master/wazo_call_logd/plugins/cdr/http.py#L218